### DISCUSSION TITLE
Wormchain should be split into a separate repo

### DISCUSSION THREAD

**Discussion by gearlbrace** [wormhole-foundation/wormhole#2447] [General] [answered]
Wormhole and Wormchain are currently sourced in the same monorepo.

Wormchain and Wormhole releases are independent projects but track on the same tags regardless of which project is updated. They should be separated to provide independent tags and releases.

**Comment by SEJeff** [comment, marked as answer]
Unfortunately, the wormhole project uses a monorepo structure. I don't love it either, but that's unlikely to change. There are [wormchain specific tags](https://github.com/wormhole-foundation/wormhole/releases/tag/v2.23.26-wormchaind) now for dedicated wormchain releases.
